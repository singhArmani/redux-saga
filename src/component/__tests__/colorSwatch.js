import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ColorSwatch from '../colorSwatch';

describe('ColorSwatch', () => {
  const testProps = {
    text: '',
  };
  it('renders correctly', () => {
    const tree = renderer.create(<Router><ColorSwatch {...testProps} /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
