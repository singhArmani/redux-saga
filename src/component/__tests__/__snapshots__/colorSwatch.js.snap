// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorSwatch renders correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": undefined,
      "borderColor": "#000000",
      "borderRadius": 15,
      "borderStyle": "solid",
      "margin": 15,
      "padding": 20,
      "textAlign": "center",
    }
  }
>
  <h2>
    Text: 
    
  </h2>
  <h2>
    Path: 
    /
  </h2>
</div>
`;
